crud/select.lua

```diff
@@ -26,6 +26,14 @@ local SELECT_FUNC_NAME = '_crud.select_on_storage'
 
 local DEFAULT_BATCH_SIZE = 100
 
+local available_modes = {
+    read = true,
+    write = true,
+}
+function checkers.call_mode(p)
+    return available_modes[p] ~= nil
```

Comment:
Maybe it's even faster to perform naive check: `mode == 'read' or mode == 'write'`